src/components/homePage/filmSearch/search/search.module.scss

```diff
@@ -8,6 +10,14 @@
   box-sizing: border-box;
 }
 
+@media (max-width:$tablet) {
+  .searchBarContainer {
+    display: flex;
+    padding: 0 20px;
+    justify-content: center;
+  }
+}
+
 // gradient animation
 @keyframes gradient {
```

Comment:
please move all thees animations to separated file animations.scss to styles. Perhaps, once you will need to use it in another component